apps/fluent-tester/src/TestComponents/Switch/E2ESwitchTest.tsx

```diff
@@ -1,19 +1,25 @@
 import * as React from 'react';
 import { Stack } from '@fluentui-react-native/stack';
 import { Switch } from '@fluentui-react-native/switch';
-import { Text } from 'react-native';
-import { stackStyle } from '../Common/styles';
+import { Text } from '@fluentui-react-native/experimental-text';
+import { stackStyle, commonTestStyles } from '../Common/styles';
 import {
   SWITCH_TEST_COMPONENT,
   SWITCH_ACCESSIBILITY_LABEL,
   SWITCH_NO_A11Y_LABEL_COMPONENT,
   SWITCH_TEST_COMPONENT_LABEL,
   SWITCH_ON_PRESS,
 } from './consts';
+import { Platform } from 'react-native';
 
 export const E2ESwitchTest: React.FunctionComponent = () => {
   const [switchPressed, setSwitchSwitchPressed] = React.useState(false);
 
+  const memoizedStyles = React.useMemo(
+    () => [Platform.OS === 'android' ? { ...commonTestStyles.androidContainer, height: 100, marginBottom: 50 } : stackStyle],
```

Comment:
nit: This doesn't need to be wrapped in an array.

Reply:
Same comment for the rest of the memoed styles

Reply:
Changed across all the memoized styles.

Reply:
Changes done and approved.